### PR TITLE
`github_repository` - adding support for `archived` repositories

### PR DESCRIPTION
**Pull request by tombuildsstuff**
- Adds support for archiving repositories
- Updates the SDK to the latest commit
- Fixing the failing deployment tests (by giving them unique names)

After some additional testing with this it appears repositories can only be archived through the API and not unarchived (whereas this is possible through the UI) - so this could do with some 🤔

### CODE REVIEW

File: website/docs/r/repository.html.markdown [hunk 1]
```diff
@@ -64,6 +64,8 @@ The following arguments are supported:
 and after a correct reference has been created for the target branch inside the repository. This means a user will have to omit this parameter from the
 initial repository creation and create the target branch inside of the repository prior to setting this attribute.
 
+* `archived` - (Optional) Specifies if the repository should be archived. Defaults to `false`.
```
Comment (catsby) [inline website/docs/r/repository.html.markdown]: Need to document the limitation found about unarchiving via the API. If you do choose to add a `CustomizeDiff` func, please document the behavior of going from `true`->`false` (unarchiving)

**Comment by paultyng** [comment]
This should fix #73

**Comment by majormoses** [comment]
Can we update the docs and get this merged?

**Comment by paultyng** [comment]
This was released in v1.1.0

**Pull request merged**